client/onboarding-prototype/steps/loading.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { addQueryArgs } from '@wordpress/url';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { useOnboardingContext } from '../context';
+import { EligibleData, EligibleResult } from '../types';
+import { fromDotNotation } from '../utils';
+
+const Loading: React.FC = () => {
+	const { data } = useOnboardingContext();
+
+	const isEligibleForPo = async () => {
+		if (
+			! data.country ||
+			! data.business_type ||
+			! data.annual_revenue ||
+			! data.go_live_timeframe
+		) {
+			return false;
+		}
+		const businessDetails: EligibleData = {
+			business: {
+				country: data.country,
+				type: data.business_type,
+				mcc: 'computers_peripherals_and_software', // TODO GH-4853 add MCC from onboarding form
+				annual_revenue: data.annual_revenue,
+				go_live_timeframe: data.go_live_timeframe,
+			},
+		};
+		const eligibleResult = await apiFetch< EligibleResult >( {
```

Comment:
I guess we probably need try/catch here, right?

Reply:
I think so, we should handle the error and show something to the merchant. With [useApiFetch](https://github.com/Automattic/woocommerce-payments/blob/28e46d6ff48a2135b97eb10d68b6052c48b33305/client/hooks/use-api-fetch.ts) I used `core/notices`, but maybe for this case we should render it directly as part of the component.

Reply:
@ismaeldcom what if we don't show the error to the merchant and we just fall back to full onboarding path? 
try/catch isEligibleForPo call and make it  isEligibleForPo = false in case of error. We'll get server error logs and will be able to see that something is not working

Reply:
🤔 That sounds good! We just must be sure that those logs are properly handled to notice any issue with that call.

Reply:
Added [here](https://github.com/Automattic/woocommerce-payments/pull/5779/commits/fe09ff8df02afca610a5c3dd6c9dbf6f88c3a78e)!